### DISCUSSION TITLE
I can't use CelebA datasets in ch12.3.5.

### DISCUSSION THREAD

**Discussion by AyanoSagasaki** [rasbt/machine-learning-book#152] [Q&A]
I'm using `celeba_dataset = torchvision.datasets.CelebA(image_path, split='train', target_type='attr', download=False)`, but I can't.

I already have read [this page](https://github.com/rasbt/machine-learning-book/blob/main/ch12/ch12_part1.ipynb), and tried. Downloaded text files and zip file from Google Drive folder, and unziped.
However, I can't use CelebA.

![image](https://github.com/rasbt/machine-learning-book/assets/114718837/0f0824c2-8181-4330-a6a2-164f95d9769f)

**Comment by rasbt** [comment]
That's strange, unfortunately, I don't have a good explanation since it worked for me. I saw someone else posting this https://github.com/rasbt/machine-learning-book/issues/146 which might help.

**Reply by AyanoSagasaki** [reply]
Thank you so much for your reply!! When I tried the method in #146 , I could use CelebA.

**Reply by pdhphd** [reply]
I'm having the same issue but I've downloaded and unzipped the file via the link in #146 and set "download=False" and now I repeatedly get the error "Dataset not found or corrupted..."

**Reply by pdhphd** [reply]
This is my output
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
Cell In[61], line 2
      1 image_path = './'
----> 2 celeba_train_dataset = torchvision.datasets.CelebA(image_path, split='train', target_type='attr', download=False)
      3 celeba_valid_dataset = torchvision.datasets.CelebA(image_path, split='valid', target_type='attr', download=False)
      4 celeba_test_dataset = torchvision.datasets.CelebA(image_path, split='test', target_type='attr', download=False)

File ~/Library/Python/3.9/lib/python/site-packages/torchvision/datasets/celeba.py:83, in CelebA.__init__(self, root, split, target_type, transform, target_transform, download)
     80     self.download()
     82 if not self._check_integrity():
---> 83     raise RuntimeError("Dataset not found or corrupted. You can use download=True to download it")
     85 split_map = {
     86     "train": 0,
     87     "valid": 1,
     88     "test": 2,
     89     "all": None,
     90 }
     91 split_ = split_map[verify_str_arg(split.lower(), "split", ("train", "valid", "test", "all"))]

RuntimeError: Dataset not found or corrupted. You can use download=True to download it

**Reply by pdhphd** [reply]
Please disregard. I moved the celeba from Chapter 12 into my directory for Chapter 14 and it worked, as it did in Chapter 12.
